### PR TITLE
Audit tables created for Invoice and Project

### PR DESCRIPTION
**Pull request by Kirannegi02**
Targets #2151 

### Description
- Added the Invoice and project audit table in database.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,10 +34,12 @@
         "laravel/tinker": "^2.0",
         "laravel/ui": "^3.0",
         "laravelcollective/html": "^6.2",
+        "laravelista/lumen-vendor-publish": "^8.0",
```
Comment (P4NK4J) [inline composer.json:37]: why we are updating this ?

**Review by P4NK4J — COMMENTED** [review]
Looks good, just left a question

File: composer.json (hunk 1 above)
Comment (Kirannegi02) [reply composer.json:37]: Actually, I have noticed it and there is no use of this.

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**